foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/OrphedomosToFabric8Migration.java

```diff
@@ -0,0 +1,277 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.util.pom.LocationAwareMavenReader;
+import org.apache.maven.model.Build;
+import org.technologybrewery.baton.util.FileUtils;
+import com.boozallen.aissemble.upgrade.util.pom.PomHelper;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.model.InputLocation;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import com.boozallen.aissemble.upgrade.util.pom.PomModifications;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.getLocationAnnotatedModel;
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.writeModifications;
+
+import static org.technologybrewery.baton.util.FileUtils.getRegExCaptureGroups;
+
+/**
+ * This migration is responsible for migrating pom files using Orphedomos to use the Fabric8 Docker Maven plugin.
+ */
+
+public class OrphedomosToFabric8Migration extends AbstractAissembleMigration{
+    private static final Logger logger = LoggerFactory.getLogger(OrphedomosToFabric8Migration.class);
+    private static final String OLD_GROUP_ID = "org.technologybrewery.orphedomos";
+    private static final String OLD_ARTIFACT_ID = "orphedomos-maven-plugin";
+    private static final String NEW_GROUP_ID = "io.fabric8";
+    private static final String NEW_ARTIFACT_ID = "docker-maven-plugin";
```

Comment:
A: I would recommend refactoring these to be like `ORPH_GROUP_ID` and `FABRIC8_GROUP_ID`

Reply:
refactored to use

```
    private static final String ORPH_GROUP_ID = "org.technologybrewery.orphedomos";
    private static final String ORPH_ARTIFACT_ID = "orphedomos-maven-plugin";
    private static final String FABRIC8_GROUP_ID = "io.fabric8";
    private static final String FABRIC8_ARTIFACT_ID = "docker-maven-plugin";
```

Reply:
No need to change here - but using whole words, even when they are long, is generally easier to read/understand than `ORPH` - I know what this PR is for and I still had to think "why ORPH" for a moment.  We should just use `OPRHEDOMOS_GROUP_ID` instead.  Rarely is there a need to shorten things.

Reply:
@d-ryan-ashcraft added the full name to variables!